projects/core/src/datetimepicker/calendar.ts

```diff
@@ -156,16 +163,24 @@ export class MatDatetimepickerCalendar<D> implements AfterContentInit, OnDestroy
   }
 
   /** Whether the calendar is in month view. */
-  _currentView: "clock" | "month" | "year" = "month";
-  _clockView: "hour" | "minute" = "hour";
+  _currentView: MatCalendarView;
```

Comment:
should be marked as `private`
```suggestion
  private _currentView: MatCalendarView;
```